keras/layers/noise.py

```diff
@@ -90,3 +90,64 @@ def get_config(self):
         config = {'rate': self.rate}
         base_config = super(GaussianDropout, self).get_config()
         return dict(list(base_config.items()) + list(config.items()))
+
+
+class AlphaDropout(Layer):
```

Comment:
Is there a reason why there is not a `SpatialAlphaDropout` like there is a `SpatialDropout`?
In the paper they are not explicitly doing it, but they do have an argument `noise_shape` on their Github. 
When they release the code for more advanced datasets, we'll know for sure I guess.